planning/freespace_planner/package.xml

```diff
@@ -4,10 +4,12 @@
   <name>freespace_planner</name>
   <version>0.1.0</version>
   <description>The freespace_planner package</description>
+  <maintainer email="kosuke.takeuchi@tier4.jp">Kosuke Takeuchi</maintainer>
   <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
+  <maintainer email="takayuki.murooka@tier4.jp">Takayuki Murooka</maintainer>
+  <maintainer email="yutaka.shimizu@tier4.jp">Yutaka Shimizu</maintainer>
   <license>Apache License 2.0</license>
 
-  <author email="kenji.miyake@tier4.jp">Kenji Miyake</author>
```

Comment:
same to https://github.com/autowarefoundation/autoware.universe/pull/2766#discussion_r1089077422